doc/nrf/ug_nrf5340.rst

```diff
@@ -630,14 +630,20 @@ This container package can be used by update tools to pass both images during th
 Getting logging output
 **********************
 
-When connected to a computer, the nRF5340 DK emulates three virtual COM ports.
+When connected to a computer, the nRF5340 DK emulates two virtual COM ports.
 In the default configuration, they are set up as follows:
 
 * The first COM port outputs the log from the network core (if available).
-* The second (middle) COM port is routed to the **P24** connector of the nRF5340 DK.
-* The third (last) COM port outputs the log from the application core sample.
+* The second COM port outputs the log from the application core sample.
 
-You can use the second COM port as follows:
+P24 connector in the nRF5340 DK v1.0.0
```

Comment:
```suggestion
Third COM port only in the v1.0.0 nRF5340 DK
```

Let's put the important information in the title, and have where it is routed to as a detail in the section.

Reply:
Is it possibly to include the information that the third vcom will appear as the second one without making this too confusing? They will appear listed in order when you attach it. So ttyACM0 => net, ttyACM1 => p24, ttyACM2 => app.

Reply:
Maybe the easiest is to just an additional section for the v1.0.0 board?

Reply:
IMHO it is currently a bit messy, but still not possible to misunderstand if the full docs are read.

Feel free to try to find a better phrasing with @greg-fer .

Reply:
I think the existing phrasing was understandable, so keeping the current phrasing as note for the v1.0.0 DK and adding an updated phrasing for the most recent version is clear to me.

```
2.0.0+ 
When connected to a computer, the nRF5340 DK emulates two virtual COM ports.
In the default configuration, they are set up as follows:

* The first COM port outputs the log from the network core (if available).
* The second COM port outputs the log from the application core.

1.0.0
When connected to a computer, the nRF5340 DK emulates three virtual COM ports.
In the default configuration, they are set up as follows:
* The first COM port outputs the log from the network core (if available).
* The second (middle) COM port is routed to the **P24** connector of the nRF5340 DK.
* The third (last) COM port outputs the log from the application core.
```

Note that I also removed the word `sample` from the sentences. It doesn't belong there I think.

Reply:
@greg-fer The above is just a suggestion, your current version also works fine. No objections.

Reply:
I like your suggestion, @joerchan . Pushed edits.